app/api/server/v1/channels.js

```diff
@@ -8,6 +8,7 @@ import { normalizeMessagesForUser } from '../../../utils/server/lib/normalizeMes
 import { API } from '../api';
 import { settings } from '../../../settings';
 import { Team } from '../../../../server/sdk';
+import { validateName } from '../../../lib';
```

Comment:
if you can, import direct from the file:
```suggestion
import { validateName } from '../../../lib/server/functions/validateName';
```